.github/actions/ginkgo/main-focus.yaml

```diff
@@ -212,6 +213,10 @@ include:
   - focus: "f20-kafka"
     cliFocus: "K8sKafkaPolicyTest"
```

Comment:
`f20-kafka` is the test that is currently taking least amount of time to run. I would suggest to add the new test here.

```
cliFocus: "K8sKafkaPolicyTest|K8sSpecificMACAddressTests"
```